CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+cmake_minimum_required(VERSION 3.3)
+project(EDM4HEP VERSION 0.1.0)
+
+#--- Run the podio class generator and link library -------
+
+find_package(podio REQUIRED HINTS $ENV{PODIO})
+
+execute_process(COMMAND python ${podio_CMAKE_DIR}/../python/podio_class_generator.py edm4hep.yaml edm4hep edm4hep
+                WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
+
+find_library(PODIOLIB podio PATHS ${podio_DIR}/../lib)
```

Comment:
The `find_library` shouldn't be needed after a `find_package`.  That probably indicates an issue with podio's "PodioConfig" file.